### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vsupruniuk**

### CODE REVIEW

**Comment by vsupruniuk** [comment]
image
image
image
image
image
image

File: templates/taxi/driver_form.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} driver</h1>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    {{ form|crispy }}
+
+    <input type="submit" value="Submit" class="btn btn-primary">
+  </form>
+{% endblock %}
```
Comment (LiudmylaKulinchenko) [inline templates/taxi/driver_form.html]: missed blank line

File: templates/taxi/driver_confirm_delete.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <button type="submit" class="btn btn-danger">Yes</button>
+    <a href="{% url "taxi:driver-detail" pk=driver.pk %}" class="btn btn-outline-danger">No</a>
+  </form>
```
Comment (LiudmylaKulinchenko) [inline templates/taxi/driver_confirm_delete.html]: Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 🔥

**Pull request closed without merging**